cache/lru_cache.h

```diff
@@ -206,13 +207,16 @@ class ALIGN_AS(CACHE_LINE_SIZE) LRUCacheShard : public CacheShard {
   double GetHighPriPoolRatio();
 
   // Overloading to aligned it to cache line size
+  // They are used by tests.
   void* operator new(size_t);
 
-  void* operator new[](size_t);
+  // placement new
+  void* operator new(size_t, void*);
 
   void operator delete(void *);
 
-  void operator delete[](void*);
+  // placement delete, does nothing.
+  void operator delete(void*, void*);
```

Comment:
Do we still need those functions?

Reply:
They are to make sure `new LRUCacheShard()` in lru_cache_test.cc:28 return aligned memory.